packages/oasis-actions/src/helpers/ajna/index.ts

```diff
@@ -112,3 +112,14 @@ export const calculateAjnaApyPerDays = (amount: BigNumber, apy: BigNumber, days:
     .times(new BigNumber(Math.E ** apy.times(days).div(365).toNumber()))
     .minus(amount)
     .div(amount)
+
+// The origination fee is calculated as the greatest of the current annualized
+// borrower interest rate divided by 52 (one week of interest) or 5 bps multiplied by the loan’s new
+// debt.
+export const getAjnaBorrowOriginationFee = ({
+  interestRate,
+  quoteAmount,
+}: {
+  interestRate: BigNumber
+  quoteAmount: BigNumber
+}) => BigNumber.max(interestRate.div(52), new BigNumber(0.0005)).times(quoteAmount)
```

Comment:
Can you name the magic numbers? 🙏

Reply:
I added description above this method to clarify what 52 and 0.0005 means. I thought about extracting it as const but I doubt if we ever reuse it since these seems to be method specific.

Reply:
I agree about naming the number. Even if it's not going to be reused, it's better to take this number and write it as a const at the beginning of the file, without exporting it. That'll make it easier to read.